docs/source/terms_and_model.rst

```diff
@@ -1147,3 +684,965 @@ The GA4GH computed identifier for these sets is
+   * - start
+     - :ref:`HumanCytoband`
+     - 1..1
+     - name of Cytoband at the interval start (see below)
+   * - end
+     - :ref:`HumanCytoband`
+     - 1..1
+     - name of Cytoband at the interval end (see below)
+
+**Implementation Guidance**
+
+* When using :ref:`CytobandInterval` to refer to human cytogentic
+  bands, the following conventions MUST be used. Bands are denoted by
+  the arm ("p" or "q") and position (e.g., "22", "22.3", or the symbolic
+  values "cen" or "ter") per ISCN conventions [1]_. These conventions
+  identify cytobands in order from the centromere towards the telomeres.
+  In VRS, we order cytoband coordinates in the p-ter → cen → q-ter
+  orientation, analogous to sequence coordinates. This has the
+  consequence that bands on the p-arm are represented in descending
+  numerical order when selecting cytobands for `start` and `end`.
+
+**Examples**
+
+.. parsed-literal::
+
+   {
+     "end": "p22.1",
+     "start": "p22.3",
+     "type": "CytobandInterval"
+   }
+
+.. _SequenceExpression:
+
+Sequence Expression
+@@@@@@@@@@@@@@@@@@@
+
+VRS provides several syntaxes for expressing a sequence,
+collectively referred to as *Sequence Expressions*. They are:
+
+* :ref:`LiteralSequence`: An explicit :ref:`Sequence`.
+* :ref:`DerivedSequence`: A sequence that is derived from a
+  :ref:`Sequencelocation`.
+* :ref:`RepeatedSequence`: A description of a repeating :ref:`Sequence`.
+
+
+.. _LiteralSequence:
+
+LiteralSequence
+###############
+
+A LiteralSequence "wraps" a string representation of a sequence for
+parallelism with other SequenceExpressions.
+
+**Computational Definition**
+
+An explicit expression of a Sequence.
+
+**Information Model**
+
+.. list-table::
+   :class: reece-wrap
+   :header-rows: 1
+   :align: left
+   :widths: auto
+
+   * - Field
+     - Type
+     - Limits
+     - Description
+   * - type
+     - string
+     - 1..1
+     - MUST be "LiteralSequence"
+   * - sequence
+     - :ref:`Sequence`
+     - 1..1
+     - The sequence to express
+
+
+.. _DerivedSequence:
+
+DerivedSequence
+###############
+
+Certain mechanisms of variation result from relocating and
+transforming sequence from another location in the genome.
+A *derived sequence* is a mechanism for expressing (typically
+large) reference subsequences specified by a :ref:`SequenceLocation`.
+
+**Computational Definition**
+
+An expression of a sequence that is derived from a referenced
+sequence location.
+
+**Information Model**
+
+.. list-table::
+   :class: reece-wrap
+   :header-rows: 1
+   :align: left
+   :widths: auto
+
+   * - Field
+     - Type
+     - Limits
+     - Description
+   * - type
+     - string
+     - 1..1
+     - MUST be "DerivedSequence"
+   * - location
+     - :ref:`SequenceLocation`
+     - 1..1
+     - The location describing the sequence
+   * - reverse_complement
+     - bool
+     - 1..1
+     - When True, indicates the derived sequence is a
+       reverse complement of the location sequence
```

Comment:
We should consider moving `reverse_complement` to SequenceLocation. I think we will want this ability when we are working on introns in transcript locations.

Reply:
We agree this needs additional discussion, we will address in #276 and remove `reverse_complement` from 1.2.0 release.

Reply:
Dropped from docs and verified that it was already removed from the schema.